src/index.js

```diff
@@ -1 +1,54 @@
 import './style.css';
+
+const playId = 'CsVjrIhDWuIKDeZ1pqQ';
+const leaderboardForm = document.querySelector('.add-score');
+
+leaderboardForm.addEventListener('submit', async (e) => {
+  e.preventDefault();
+  const person = document.getElementById('input-name');
+  const score = document.getElementById('input-score');
+  const leaderboardFormData = {
+    person: person.value,
+    score: parseInt(score.value, 10),
+  };
+  const response = await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${playId}/scores/`, {
+    method: 'POST',
+    body: JSON.stringify(leaderboardFormData),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+  const result = await response.json();
+  person.value = '';
+  score.value = '';
+  return result;
+});
```

Comment:
- [ ] I added my name and score, hit the submit button, and then clicked on the refresh button, but my entries were not added to the scoreboard. Kindly fix this so users can see their entries on the scoreboard after creating them and clicking the refresh button.


- [ ] Also, when I opened your project, I saw the previous user's entry, but their names were showing `undefined` kindly fix this so that the user's name shows instead of undefined.

<img width="695" alt="chrome_u4jFgcT2ze" src="https://user-images.githubusercontent.com/81468675/180219575-cf25f3b8-493d-42b5-9c1e-ae6a116fe8ab.png">

Reply:
Thanks, a lot @julie-ify